app/detail mock.html

```
@@ -0,0 +1,139 @@
+                        <span class="material-symbols-outlined text-sm">signal_cellular_alt</span>
+                        <span class="material-symbols-outlined text-sm">wifi</span>
+                        <span class="material-symbols-outlined text-sm">battery_full</span>
+                </div>
+        </div>
+        <header class="px-6 pt-2 pb-4 border-b border-gray-100 bg-white">
+                <div class="flex items-center gap-1 text-sm font-medium text-blue-600 mb-1">
+                        <span class="material-symbols-outlined text-sm">folder</span>
+                        <span>Marketing</span>
+                </div>
+                <div class="mb-4">
+                        <input class="w-full text-2xl font-bold border-none p-0 focus:ring-0 placeholder-gray-300"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)